ffi/Makefile

```diff
@@ -0,0 +1,29 @@
+RUSTTARGET ?= ./target
+PREFIX ?= /usr/local
+
+CARGO ?= cargo
+SCDOC ?= scdoc
+DOXYGEN ?= doxygen
+LIBDIR ?= lib
+INCDIR ?= include
+
+all: coupe
+
+coupe:
+	$(CARGO) build --locked --release
+
+doc:
+	$(DOXYGEN) Doxyfile
+	xdg-open build/html/index.html
```

Comment:
I don't know if people use partitioners on windows, this wouldn't work on such OS.

Maybe it's better to add a note in the readme